<launch>
    <arg name="config_file" default="$(find object_pose_utils)/configs/asus_surgical_config01.xml"/>
	<arg name="board_frame" default="board_frame" />
    <arg name="thresh_block_size" default="31" />
    <arg name="thresh_const" default="10" />
    <arg name="camera"       default="/r2/head/asus/rgb/image_raw" />
    <arg name="camera_info"  default="/r2/head/asus/rgb/camera_info" />

    <node name="board_transform" pkg="tf" type="static_transform_publisher" args=".03 .04 0 -1.57 0 0 ar_marker_0 $(arg board_frame) 100"/>

    <node name="tagged_object_masker" pkg="object_pose_utils" type="tagged_object_masker" args="$(arg config_file)" output="screen">
        <param name="board_frame" value="$(arg board_frame)"/>
        <param name="thresh_block_size" value="$(arg thresh_block_size)"/>
        <param name="thresh_const" value="$(arg thresh_const)"/>
        <param name="board_frame" value="$(arg board_frame)"/>
        <remap from="in_image" to="$(arg camera)" />
        <remap from="in_camera_info" to="$(arg camera_info)" />
    </node>

</launch>
